feature-pack/pom.xml

```diff
@@ -1139,12 +1139,6 @@
         <dependency>
             <groupId>org.jboss.resteasy</groupId>
             <artifactId>resteasy-json-p-provider</artifactId>
-            <exclusions>
```

Comment:
this execlusion needs to stay, otherwise build fails with

```
[03:24:50] :	 [Step 2/3] org.wildfly:wildfly-feature-pack:pom:11.0.0.Alpha1-SNAPSHOT
[03:24:50] :	 [Step 2/3]    org.jboss.resteasy:resteasy-json-p-provider:jar:3.0.21.Final:compile has transitive dependencies:
[03:24:50] :	 [Step 2/3]       org.glassfish:javax.json:jar:1.0.4:compile
```